scalableminds/webknossos
# improve element class guess for multi-channel datalayer

- also look at voxel size when determining Element Class
### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- press `Edit` on a dataset that has a multi-channel datalayer
- wK should guess correct element class 

### Issues:
- fixes #4086 

------
